test/integration-serial/serial_deploy_test.rb

```diff
@@ -102,16 +102,6 @@ def test_multiple_configuration_files
     ], in_order: true)
     reset_logger
 
-    ENV['KUBECONFIG'] = nil
```

Comment:
I think we should modify this test to show that `ENV['KUBECONFIG'] = nil` results is the new default behavior

Reply:
Just `assert_deploy_success(result)`, or something else?

Reply:
Yeah, I think that would do it. Thought it's probably worth pulling out into its own test. Instead of being in this long one

Reply:
I was working on the assumption that all the other tests would fail if it were not set. Is that not true.

Reply:
They will, but I'd still like to have an explicit assertion.

Reply:
Done. Could I get another review?